test/remote/gateways/remote_creditcall_test.rb

```diff
@@ -147,15 +147,15 @@ def test_failed_verify
     @declined_card.number = ''
     response = @gateway.verify(@declined_card, @options)
     assert_failure response
-    assert_match %r{PAN Must be >= 13 Digits}, response.message
+    assert_match %r{PAN Must be >= 12 Digits}, response.message
```

Comment:
Both of the changes in this file are just updated error messaging from creditcall.